cluster.yml

```diff
@@ -30,15 +30,24 @@
     - { role: docker, tags: docker }
     - { role: rkt, tags: rkt, when: "'rkt' in [ etcd_deployment_type, kubelet_deployment_type ]" }
 
-- hosts: etcd:!k8s-cluster
+- hosts: all
+  any_errors_fatal: true
+  roles:
+    - { role: vault, tags: vault, vault_bootstrap: true, when: "cert_management == 'vault'" }
+
+- hosts: etcd:k8s-cluster
```

Comment:
@bogdando Intentional change, actually. In the original cluster.yml, you run the etcd role against `etcd:!k8s-cluster`, then immediately run it against `k8s-cluster` (the line removed below). I assume you need to run against the `etcd` group by itself first to setup the cluster, so the `k8s-cluster` can setup etcd-proxy later? It'd be nice if the etcd role could handle that in a single pass, but I can keep them separate for now.